crates/gourgeist/src/bare.rs

```diff
@@ -157,6 +165,7 @@ pub fn create_bare_venv(location: &Utf8Path, interpreter: &Interpreter) -> io::R
         let activator = template
             .replace("{{ VIRTUAL_ENV_DIR }}", location.as_str())
             .replace("{{ BIN_NAME }}", bin_name)
+            .replace("{{ PROMPT }}", prompt)
```

Comment:
nit: use `VIRTUAL_PROMPT` to match the original script. You'll need to update the references in all of the activation scripts.

https://github.com/pypa/virtualenv/blob/fa283474fd199e3836f8b2c99510190c6b77e2bc/src/virtualenv/activation/bash/activate.sh#L58

```suggestion
            .replace("{{ VIRTUAL_PROMPT }}", prompt)
```